runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -880,6 +887,56 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 		addLoot(event, -1, LootRecordType.EVENT, metadata, items);
 	}
 
+	private boolean handleWildernessLootKeys()
+	{
+		boolean recordedLoot = false;
+
+		for (int containerId : PVP_LOOT_KEY_CONTAINERS)
+		{
+			final Collection<ItemStack> items = toItemStacks(client.getItemContainer(containerId));
+			if (items.isEmpty())
+			{
+				continue;
+			}
+
+			addLoot("Loot key", -1, LootRecordType.EVENT, null, items);
```

Comment:
Never mind, "Loot Chest" is how we recorded pvp loot before. We'll keep the name.
```suggestion
			addLoot("Loot Chest", -1, LootRecordType.EVENT, null, items);
```